hq_superset/utils.py

```diff
@@ -16,6 +16,10 @@
 ASYNC_DATASOURCE_IMPORT_LIMIT_IN_BYTES = 5_000_000  # ~5MB
 
 
+class CCHQApiException(Exception):
```

Comment:
+1 for this change, although what do you think about adding this to an `exceptions.py` file? I don't have strong feelings either way, just wanting to gauge your thoughts...